WordCount.awk

```diff
@@ -0,0 +1,8 @@
+{
+ for (i = 1; i <= NF; i++)
+ freq[$i]++
+}
+END {
+ for (word in freq)
+ printf "%s\t%d\n", word, freq[word]
+}
```

Comment:
:scratches head:  what is this?  and what language?  and what does it have to do with babynames?